PrestaShopCorp/ps_checkout
# Release 1.4.0

Please go to `Checks` subtab here, then on `Build & Release draft` to download zip

### How to test
Everything should work as before, with this changes :

- If a payment is declined by PayPal, no more Order with Order State Payment error is created
- After a successful order, transaction id should always set in BO > Order view > Payment block
- On all orders paid with PrestaShop Checkout, a new block should be displayed on BO > Order view > PrestaShop Checkout (With transaction details and refund form)
- If a locale used in PrestaShop is not supported by PayPal, no more error on checkout when PayPal Sdk is loaded.
- If merchant is not authorized to use Hosted Fields, no display it in payment options on checkout and in BO > PrestaShop Checkout configuration page an explanation is displayed
- New user guide pdf available in BO > PrestaShop Checkout configuration > Help for NL, PL, PT
- Support of OrderState outofstock : see bellow

#### How to test OrderState outofstock
- Create a Cart with a product outofstock but allowed to be ordered.
- Go to Proceed to checkout
- Choose to paid with PrestaShop Checkout
- Paid
- Go to BO > Orders
- See OrderState `On backorder (paid)` instead of `Payment accepted`

For new refund process please refer to user guide in BO > PrestaShop Checkout configuration > Help
